lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
The problem with having the mixins subclass directly from `SharedModuleStoreTestCase` is that all test methods defined on the mixins would be executed.  For example, the tests in `CourseDetailTestMixin` aren't meant to be tested on the abstract mixin class.